internal/rpc/rpc.go

```diff
@@ -170,6 +171,10 @@ func (c *Command) post(method string, req interface{}) error {
 	}
 
 	r.Header.Set("Content-Type", "application/json")
+	if c.UserAgent != "" {
+		r.Header.Set("User-Agent", c.UserAgent)
```

Comment:
Since this is hardcoded anyway - could we just hardcode it here and minimize the changes, instead of passing it through and adding a check here?

Reply:
That seems fine to me!